backup/setup/migrations.json

```diff
@@ -369,187 +369,198 @@
 },
 {
 	"_id": {
-		"$oid": "5e4a61ca1db4f62c8cb2b9df"
+		"$oid": "5e970516325e804198ec63dd"
 	},
 	"state": "up",
-	"name": "remove_invalid_accounts",
+	"name": "addAdminPermissionForManagingStudentTeamCreation",
 	"createdAt": {
-		"$date": "2020-02-17T09:50:02.639Z"
+		"$date": "2020-04-15T12:59:02.81Z"
 	},
 	"__v": 0
 },
 {
 	"_id": {
-		"$oid": "5e4d449dc8fe4e2ff4993849"
+		"$oid": "5e98318cfb423a82b448ce99"
 	},
 	"state": "up",
-	"name": "datasource-permissions-for-superhero",
+	"name": "add-sync-start-permission",
 	"createdAt": {
```

Comment:
please only add the changed line an keep the old ones, otherweise others will have merge conflicts

Reply:
done